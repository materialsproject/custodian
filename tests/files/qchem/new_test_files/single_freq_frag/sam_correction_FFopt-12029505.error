/global/home/users/rdguha/miniconda3/envs/rg_cup/lib/python3.9/site-packages/paramiko/transport.py:236: CryptographyDeprecationWarning: Blowfish has been deprecated
  "class": algorithms.Blowfish,
