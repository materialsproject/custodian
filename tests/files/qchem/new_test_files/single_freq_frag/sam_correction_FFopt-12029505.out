2022-06-07 21:00:24,111 INFO Hostname/IP lookup (this will take a few seconds)
2022-06-07 21:00:24,112 INFO Launching Rocket
2022-06-07 21:00:27,685 INFO RUNNING fw_id: 340 in directory: /global/scratch/users/rdguha/hydr_halide/frag_halide/block_2022-06-08-03-16-58-295470/launcher_2022-06-08-04-00-17-629823
2022-06-07 21:00:27,736 INFO Task started: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}}.
2022-06-07 21:00:27,741 INFO Task completed: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}}
2022-06-07 21:00:27,754 INFO Task started: {{atomate.qchem.firetasks.run_calc.RunQChemCustodian}}.
